packages/contentful/src/mocks/graphql-client.mock.ts

```diff
@@ -58,7 +58,9 @@ export const getContentfulGraphqlClientMockServer = (
 ): GraphQLClient => {
   const schemaLocation = '../crn/schema/autogenerated-schema.graphql';
   const baseMocks = {
+    AnnouncementCollection: resultDto,
```

Comment:
Shouldn't it be
```suggestion
    AnnouncementsCollection: resultDto,
```

because we have this in `crn/schema/autogenerated-schema.graphql`

<img width="1084" alt="Screenshot 2023-12-11 at 14 20 33" src="https://github.com/yldio/asap-hub/assets/16595804/7db03670-84ae-4e87-be98-810dcbdd890f">